content/docs/engine-dev/assets/metadata.md

```diff
@@ -0,0 +1,117 @@
+There are serveral unit tests for each system.  Most can be found in these locations:
+
+* Code\Tools\AssetProcessor\native\tests\assetmanager\
+    * DelayRelocationTests
+    * AssetProcessorManagerTest
+    * SourceDependencyTests
+* Code\Tools\AssetProcessor\native\tests\UuidManagerTests
+* Code\Framework\AzToolsFramework\Tests\Metadata\
+    * MetadataManagerTests
+    * UuidUtilTests
+
+## File Support
+
+Below is a rough list of most filetypes supported by the engine categorized by relocatability based on analysis of their dependencies.  Testing and deeper analysis may reveal some of these types require more work to be fully supported.
+
+### Files which may already work
+All references to these files appear to be done via AssetID.  The sub IDs generated for product assets of these source assets appear to be stable and not based on the source asset names. 
+
+* animgraph
+* azasset
+* dat
+* emfxworkspace
+* exr
+* filetag
+* inputbindings
+* motionset
+* names
+* pass
+* pem
+* physxconfiguration
+* physxmaterial
+* postfxlayercategories
+* precompiledshader
+* resourcepool
+* scriptcanvas
+* scriptevents
+* stars
+* surfacetagnamelist
+* tfx
+* ts
+* uicanvas
+* vegdescriptorlist
+* setreg
+
+### Files which require some work to support
+Fully supporting asset relocation for assets in this category will require some improvements to the usage of these assets. Work to support files in this category can include: Stabilizing product asset sub ID, so that when a file is renamed, the sub IDs of product assets don't change, and changing references to these assets (or their product assets) from other assets to use asset IDs instead of relative paths.
+
+* azshadervariant
+* shader, materialpipeline, template
+* material
+* prefab
+* fbx
+* font, fontfamily
+* ttf, otf
+* sprite
+* texture file types (jpg, png, dds, tif, etc)
+
+### Unknown
+These are files which need further analysis to determine if support is useful or feasible and how much work, if any, is required to fully support them
+* materialtype - referenced by FBX files - suspect this is a code based dependency
+* shader, attimage - referenced by binary pass file and shadervariantlist
+* tfxbone, tfxmesh - referenced by binary tfx file
+* pak - may be referenced by code? legacy asset
+* shadervariantlist - referenced by shadervariantlist.  There is a relative path to a shader file, JSON format
+
+### Too generic
+These file extensions tend to be used by a variety of different types of content, which means that a file extension based approach to supporting relocation won't work as cleanly here. XML files, for example, can describe multiple different types of content, some of which will handle asset relocation easier than others. The easiest path forward to supporting relocation for this kind of content is to get content types to use specific file extensions instead of these generic ones. For example, even though prefab files are in the json format, they do not have the json extension.

```

Comment:
```suggestion
These file extensions tend to be used by a variety of different types of content, which means that a file extension based approach to supporting relocation won't work as cleanly here. XML files, for example, can describe multiple different types of content, some of which will handle asset relocation easier than others. The easiest path forward to supporting relocation for this kind of content is to get content types to use specific file extensions instead of these generic ones. For example, even though prefab files are in the JSON format, they do not have the JSON extension.

```